### PR TITLE
HGI-6533 / Add entity filters

### PR DESCRIPTION
**Pull request by arilton**
# Description of change
(write a short description or paste a link to JIRA)

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_chargebee/streams/base.py [hunk 1]
```diff
@@ -199,9 +197,16 @@ def sync_data(self):
         else:
             params = {"updated_at[after]": bookmark_date_posix, "updated_at[before]": self.START_TIMESTAP}
             bookmark_key = 'updated_at'
-
+        
         LOGGER.info("Querying {} starting at {}".format(table, bookmark_date))
 
+        if self.config.get("filters"):
+            entity_filters: dict[str,str] = self.config.get("filters", {}).get(self.ENTITY)
```
Comment (butkeraites-hotglue) [inline tap_chargebee/streams/base.py:204]: `self.ENTITY` is not defined for `BaseChargebeeStream`.

File: tap_chargebee/streams/base.py [hunk 2]
```diff
@@ -199,9 +197,16 @@ def sync_data(self):
...
+            if entity_filters:
```
Comment (butkeraites-hotglue) [inline tap_chargebee/streams/base.py:205]: If you define `entity_filters` as an empty dict you would be able to make this if block one indentation level lower than what it's now

**Review by butkeraites-hotglue — CHANGES_REQUESTED** [review]
Great job @arilton ! I would just define `ENTITY` for the base class and make a little change to drop one indentation lvl on the code.

**Review by butkeraites-hotglue — APPROVED** [review]
Well done!

**Review by hsyyid — APPROVED** [review]

**Pull request merged**